go/private/mode.bzl

```diff
@@ -58,34 +58,18 @@ def mode_string(mode):
         result.extend(mode.gc_goopts)
     return "_".join(result)
 
-def _ternary(*values):
-    for v in values:
-        if v == None:
-            continue
-        if type(v) == "bool":
-            return v
-        if type(v) != "string":
-            fail("Invalid value type {}".format(type(v)))
-        v = v.lower()
-        if v == "on":
-            return True
-        if v == "off":
-            return False
-        if v == "auto":
-            continue
-        fail("Invalid value {}".format(v))
-    fail("_ternary failed to produce a final result from {}".format(values))
-
 def get_mode(ctx, go_toolchain, cgo_context_info, go_config_info):
-    static = _ternary(go_config_info.static if go_config_info else "off")
+    static = go_config_info.static if go_config_info else False
```

Comment:
Why is this a safe change?

Reply:
I believe these values come from `//go:config:static` so they are already bools, though it's certainly possible that I am not fully following how this works. Tests seem to pass, not sure how our coverage is